homeassistant_cli/const.py

```diff
@@ -6,5 +6,6 @@
 REQUIRED_PYTHON_VER = (3, 5, 3)
 
 DEFAULT_SERVER = 'http://localhost:8123'
+DEFAULT_HASSIO_SERVER = 'http://hassio.local:8123'
```

Comment:
This is incorrect. Must be this:

```suggestion
DEFAULT_HASSIO_SERVER = 'http://hassio/homeassistant'
```

Reply:
yes, I was just thinking most would be using this *externally* from hass.io - but your last comment on #23 made me realize that outside hass.io there is no real good default as it could be either http or https. Updated the PR to fallback on http://hasio/homeassistant.